ui/package.json

```diff
@@ -32,6 +32,7 @@
     "react-scroll": "^1.7.16",
     "react-searchkit": "^0.18.0",
     "react-show-more": "^2.0.0",
+    "react-social-login-buttons": "^3.0.0",
```

Comment:
honestly, I would really avoid this. It is `423 kB` lib just for one button...
I would just create a black button `Sign in with GitHub` and that's it. If anyone wants to add this, will do in the overlay...

Reply:
:+1: 
I tried different libraries and they all were not very good, I will remove it